GUIDELINES.md

```diff
@@ -109,10 +109,37 @@ the `meta.yaml` files in above examples).
 In general, standard `make` should work. Other build tools (e.g., `autoconf`)
 and compilers (e.g., `gcc`) should be specified in the build requirements.
 
+We have not yet decided whether to have `gcc` as a conda package or to assume
+it is in the build environment. Until this decision is made, please add `gcc`
+(for Linux packages) and `llvm` (for OSX packages) to the `meta.yaml` as
+follows:
+
+```yaml
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+
+  run:
+    - libgcc    # [not osx]
```

Comment:
Is there a rule of thumb to figure out if libgcc is needed? Maybe the strategy should be to first try without libgcc, then if tests fail then add it back in?